install/helm/gloo/generate/values.go

```diff
@@ -240,20 +240,26 @@ type Gloo struct {
+	RestXdsPort              *uint32                 `json:"restXdsPort,omitempty" desc:"port where gloo serves REST xDS API to Envoy"`
+	ValidationPort           *int                    `json:"validationPort,omitempty" desc:"port where gloo serves gRPC Proxy Validation to Gateway"`
+	ProxyDebugPort           *int                    `json:"proxyDebugPort,omitempty" desc:"port where gloo serves gRPC Proxy contents to glooctl"`
+	Stats                    *Stats                  `json:"stats,omitempty" desc:"overrides for prometheus stats published by the gloo pod"`
+	FloatingUserId           *bool                   `json:"floatingUserId,omitempty" desc:"If true, allows the cluster to dynamically assign a user ID for the processes running in the container. If containerSecurityContext is defined, this value is not applied."`
+	RunAsUser                *float64                `json:"runAsUser,omitempty" desc:"Explicitly set the user ID for the processes in the container to run as. Default is 10101. If containerSecurityContext is defined, this value is not applied."`
+	ExternalTrafficPolicy    *string                 `json:"externalTrafficPolicy,omitempty" desc:"Set the external traffic policy on the gloo service"`
+	ExtraGlooLabels          map[string]string       `json:"extraGlooLabels,omitempty" desc:"Optional extra key-value pairs to add to the spec.template.metadata.labels data of the primary gloo deployment."`
+	ExtraGlooAnnotations     map[string]string       `json:"extraGlooAnnotations,omitempty" desc:"Optional extra key-value pairs to add to the spec.template.metadata.annotations data of the primary gloo deployment."`
+	LivenessProbeEnabled     *bool                   `json:"livenessProbeEnabled,omitempty" desc:"Set to true to enable a liveness probe for Gloo Edge (default is false)."`
+	OssImageTag              *string                 `json:"ossImageTag,omitempty" desc:"Used for debugging. The version of Gloo OSS that the current version of Gloo Enterprise was built with."`
+	ContainerSecurityContext *appsv1.SecurityContext `json:"containerSecurityContext,omitempty" desc:"SecurityContext for the gloo container. If this is defined it supercedes any values set in FloatingUserId, RunAsUser, DisableNetBind, RunUnprivileged. https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#securitycontext-v1-core"`
 	*DeploymentSpec
+	*GlooDeploymentContainer
+}
+
+type GlooDeploymentContainer struct {
+	Image                        *Image                  `json:"image,omitempty"`
+	GlooContainerSecurityContext *appsv1.SecurityContext `json:"glooContainerSecurityContext,omitempty" desc:"securitySpec for gloo container in the gloo-deployment chart. If this is defined it supercedes any values set in FloatingUserId, RunAsUser, DisableNetBind, RunUnprivileged. https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#securitycontext-v1-core"`
```

Comment:
```suggestion
	GlooContainerSecurityContext *appsv1.SecurityContext `json:"glooContainerSecurityContext,omitempty" desc:"SecurityContext for gloo container in the gloo-deployment chart. If this is defined it supercedes any values set in FloatingUserId, RunAsUser, DisableNetBind, RunUnprivileged. https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#securitycontext-v1-core"`
```
let's use the struct name `SecurityContext` since securitySpec isn't an actual field or struct name. Can you apply same changes to all instances below too?

Reply:
Not using this change in favor of just replacing them all in 1 commit, but done.